enacts/config-nma.yaml

```diff
@@ -20,13 +20,19 @@ shapes_adm:
       is_checked: False
 
 # Daily ENACTS Zarr conversion
-rr_mrg_nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/
-rr_mrg_resolution: .25
-rr_mrg_chunks:
+rr_mrg_zarr_path: /data/aaron/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL-staging/ #this I could delete and we could use the new file path defs in the maprooms?
+nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/ #top level directory path to find input data
+resolution: .0375
+chunks:
     T: 578
     Y: 25
     X: 31
-rr_mrg_zarr_path: /data/aaron/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL-staging/
+zarr_path: /data/drewr/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/ #top level directory path to put output file
```

Comment:
I don't understand why there are two zarr paths. Is that what you meant by your comment "this I could delete..." above? If rr_mrg_zarr_path is unused then definitely get rid of it.

Reply:
I really think the path of the native resolution ENACTS data should be global, not page-specific. If certain pages need parts of their calculations done ahead of time for speed reasons (e.g. regridding in this case), those can be page-specific.

But my first thought here to wonder whether we really need to do the regridding ahead of time. The Ingrid version does the regridding on the fly, so maybe we can do that here too, and avoid the complexity of having multiple copies of the data at different resolutions. @drewmresnick are you up to trying that?

Reply:
Then we should first try if no regridding at all is acceptable for the Ethiopia Onset maproom first. The regridding in Ingrid was only for performance issue. That test might have to be done on a machine more performant than devi to be relevant.

Reply:
We did try the onset calculation at the native resolution, and that was too slow. There have been some other performance improvements since then, so I guess it's worth another try.

Reply:
I am unclear exactly what you are asking me to do @aaron-kaplan. Do you want me to work to create a function that can be applied to the data on-the-fly inside of a maproom instead of running the regridding on the data during conversion such as it is inside this file?

Reply:
That's what I was suggesting, yes. But then Rémi made a different suggestion, which was to try not regridding at all, just use the dataset's native resolution in the onset maproom. I don't know if that will work, but it's easy to try, so let's try it.